packages/app/src/composables/useTransaction.ts

```diff
@@ -277,19 +412,16 @@ export function mapTransaction(
     isL1BatchSealed: transaction.isL1BatchSealed,
     error: transaction.error,
     revertReason: transaction.revertReason,
-
-    logs: logs.map((item) => ({
-      address: item.address,
-      blockNumber: item.blockNumber,
-      data: item.data,
-      logIndex: item.logIndex.toString(16),
-      topics: item.topics,
-      transactionHash: item.transactionHash!,
-      transactionIndex: item.transactionIndex.toString(16),
+    logs: logs.map((log) => ({
+      address: log.address,
+      blockNumber: log.blockNumber,
+      data: log.data,
+      logIndex: log.logIndex.toString(),
+      topics: log.topics as string[],
+      transactionHash: log.transactionHash || "",
+      transactionIndex: log.transactionIndex.toString(),
```

Comment:
Why is this block changed at all and the string representation is changed from hex-decimal to decimal?